lib/middleware/user-context.js

```diff
@@ -2,6 +2,7 @@
 
 const debug = require('debug')('loopback:componenet:access:context');
 const loopback = require('loopback');
+var Promise = require("bluebird");
```

Comment:
This should use `const` instead of var since it's value never changes. This is causing the lint test to fail:

https://circleci.com/gh/fullcube/loopback-component-access-groups/31